Products/ZenUtils/Executor.py

```diff
@@ -245,38 +266,47 @@ class ExecutorTask(object):
     @ivar call: The callable called when the task is invoked.
     """
 
-    def __init__(self, deferred, call, timeout, log):
+    def __init__(self, deferred, call, timeout, log, label=None):
```

Comment:
Probably we could assign "" as a default value for the label to exclude the if statement below

Reply:
I can do that.